docs/users/cli-usage.md

```diff
@@ -134,6 +132,11 @@ ddev config
 
 _Note: ddev config will prompt you for a project name, docroot, and project type._
 
+* the project name will be part of the URL, so make sure to avoid whitespace, underscores and special characters to avoid problems down the road.
```

Comment:
This looks like this was a regression from here https://github.com/drud/ddev/issues/86#issuecomment-370541272. Let's keep this change for now, but it will be unnecessary once we have project name validation back in place!